src/cli_common.c

```diff
@@ -427,9 +430,15 @@ sds cliVersion(void) {
 }
 
 /* This is a wrapper to call valkeyConnect or valkeyConnectWithTimeout. */
-valkeyContext *valkeyConnectWrapper(const char *ip, int port, const struct timeval tv, int nonblock) {
+valkeyContext *valkeyConnectWrapper(const char *ip, int port, const struct timeval tv, int nonblock, int rdma) {
     valkeyOptions options = {0};
-    VALKEY_OPTIONS_SET_TCP(&options, ip, port);
+    if (rdma) {
+#ifdef USE_RDMA
+        VALKEY_OPTIONS_SET_RDMA(&options, ip, port);
+#endif
```

Comment:
If `rdma` is passed when this is compiled without USE_RDMA, we shouldn't raise an error instead of ignoring it.

Let's do some assert or just return NULL?

```suggestion
#else
        assert(!rdma);
#endif
```